tests/stubdata/input/jats_aip_key_headings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE article
```

Comment:
possible to pretty print format this? I typically use https://jsonformatter.org/xml-formatter